ui/src/views/Tasks/TaskGroup/index.jsx

```diff
@@ -448,8 +476,28 @@ export default class TaskGroup extends Component {
         },
       });
 
-      this.setState({
-        taskGroupInfo: sealTaskGroup,
+      this.setState({ taskGroupInfo: sealTaskGroup });
+      this.handleSnackbarOpen({
+        message: 'Task Group sealed',
+        open: true,
+      });
+
+      return null;
+    }
+
+    if (name === 'cancelTaskGroup') {
+      const {
+        data: { cancelTaskGroup },
+      } = await apolloClient.mutate({
+        mutation: cancelTaskGroupQuery,
+        variables: {
+          taskGroupId: taskGroup.taskGroup.taskGroupId,
+        },
+      });
+
+      this.handleSnackbarOpen({
+        message: `Tasks cancelled: ${cancelTaskGroup.cancelledCount} out of ${cancelTaskGroup.taskGroupSize}.`,
```

Comment:
I wonder if the case where `cancelTaskGroup.cancelledCount < cancelTaskGroup.taskGroupSize`, additional info could be added into the message letting the user know they can try calling again to cancel remaining tasks.

Reply:
So if this happens, it probably means that some tasks were already resolved, or task group was already cancelled. As `cancelTaskGroup` would only cancel non-resolved ones. 
And we already show this in a message before user clicks the action button 

https://github.com/taskcluster/taskcluster/blob/67139383d1db65eb27f2eaeeaf2f100d69fd8e90/ui/src/views/Tasks/TaskGroup/index.jsx#L83-L87